umbra-js/test/Umbra.test.ts

```diff
@@ -343,6 +372,379 @@ describe('Umbra class', () => {
     });
   });
 
+  describe('BatchSend, scan, and withdraw funds', () => {
```

Comment:
It looks like this is *a lot* of duplication from the `describe('Send, scan, and withdraw funds'` block of tests. But we don't need to re-test scanning and withdrawing, so let's simplify things significantly and only test the new functionality here: the `batchSend` method. Duplicating this much code makes it hard to know where the important new code/tests are vs. the duplicated code/tests that aren't really adding value

Reply:
Added batchSend tests this way:
https://github.com/ScopeLift/umbra-protocol/pull/426/commits/e9a1c6dce86c129374bc9161bae7dfc219f17e7d
https://github.com/ScopeLift/umbra-protocol/pull/426/commits/18829bbc7123331418d3ab4ca33733d129f850d3
Does this work?

Reply:
Definitely looking better! Marking this resolved in favor of the next round of comments, we're almost finished here!